probe_scraper/runner.py

```diff
@@ -310,15 +311,26 @@ def sync_output_and_cache_dirs(output_bucket, cache_bucket, out_dir, cache_dir,
         sys.exit(1)
     else:
         print("Syncing output dir {}/ with s3://{}/".format(out_dir, output_bucket))
-        # The cloudfront dist will automatically gzip objects. Upload to s3.
-        sync_cft_cmd = (
-            f"aws s3 sync {out_dir}/ s3://{output_bucket}/ "
-            f"--delete "
-            f"--content-type 'application/json' "
-            f"--cache-control 'max-age=28800' "
-            f"--acl public-read"
-        )
-        os.system(sync_cft_cmd)
+
+        # cloudfront is supposed to automatically gzip objects, but it won't do that
+        # if the object size is > 10 megabytes (https://webmasters.stackexchange.com/a/111734)
+        # which our files sometimes are. to work around this, we'll regzip the contents into a
+        # temporary directory, and upload that with a special content encoding
+        with tempfile.TemporaryDirectory() as tmpdirname:
+            for filename in os.listdir(out_dir):
+                with open(f'{out_dir}/{filename}', 'rb') as f1:
+                    with gzip.open(f'{tmpdirname}/{filename}', 'wb') as f2:
+                        f2.write(f1.read())
```

Comment:
Do we want to check the size before gzipping and only gzip the big files?

Also, consistent quoting would be good here.

Reply:
I would vote that we just compress everything, as doing it selectively would require a lot of extra code. What do you mean by consistent quoting?

Reply:
Okay, I didn't know if cloudfront was fine zipping everything. Sounds good. Above you use single quotes and below you use double quotes. It's a nitpick.

Reply:
Gotcha! Easy to fix. We should probably use black to enforce this sort of thing...